src/wazuh_db/main.c

```diff
@@ -115,12 +115,14 @@ int main(int argc, char ** argv)
     }
 
     // Reset template. Basically, remove queue/db/.template.db
-    // The prefix is needed here, because we are not yet chrooted
+    // The prefix is needed here
     char path_template[OS_FLSIZE + 1];
     snprintf(path_template, sizeof(path_template), "%s/%s/%s", home_path, WDB2_DIR, WDB_PROF_NAME);
     unlink(path_template);
     mdebug1("Template file removed: %s", path_template);
 
+    os_free(home_path);
```

Comment:
I think this free shouldn't be here since `home_path` is used again later in https://github.com/wazuh/wazuh/pull/9208/files#diff-c89cdcc0b3acfda8e5e0e8bcad2ea3df1981c99d47d50243872e739b7bf9c111R225.

Reply:
I agree